PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -95,11 +95,23 @@ def nanoAOD_customizeMC(process):
 
 ### Era dependent customization
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
-#remove stuff 
+#remove stuff
 _80x_sequence = nanoSequence.copy()
 _80x_sequence.remove(isoTrackTable)
 _80x_sequence.remove(isoTrackSequence)
+_80x_sequenceMC = nanoSequenceMC.copy()
+_80x_sequenceMC.remove(genSubJetAK8Table)
 run2_miniAOD_80XLegacy.toReplaceWith( nanoSequence, _80x_sequence)
+run2_miniAOD_80XLegacy.toReplaceWith( nanoSequenceMC, _80x_sequenceMC)
 
 	
 
+from Configuration.Eras.Modifier_run2_nanoAOD_92X_cff import run2_nanoAOD_92X
+#remove stuff
+
+_92x_sequence = nanoSequence.copy()
+_92x_sequence.remove(isoTrackTable)
```

Comment:
do we need this for isoTrack?  @gpetruc

Reply:
@rappoccio can you please remove the line removing isoTrackTable?

Reply:
no, isoTrack should stay IN in 92X (we remove it only from 80X).